### DISCUSSION TITLE
Metabolic exposure in PK-sim

### DISCUSSION THREAD

**Discussion by jyna6** [Open-Systems-Pharmacology/Forum#1717] [Questions & Problems] [answered]
I'm now building a PBPK model using parent drug and its metabolites.
The simulation results shows too low metabolic exposure.
How can I make my metabolic ratio go higher in PK-sim?

**Comment by Yuri05** [comment, marked as answer]
To increase the metabolic ratio in PK-Sim, you can consider the following approaches:

1. **Adjust Enzyme Parameters:** 
   - Modify the enzyme parameters such as `Vmax `and `Km `for the metabolizing enzymes involved in the biotransformation of the parent drug to its metabolites. Increasing Vmax or decreasing Km can lead to higher metabolic rates.

2. **Incorporate Inducers/Inhibitors:**
   - Introduce inducers or inhibitors of specific enzymes involved in the metabolism of the parent drug. This can be achieved by adjusting the relevant parameters or adding specific drug-drug interaction models.

3. **Consider Physiological Changes:**
   - Incorporate any physiological changes or disease states that may affect the activity of metabolic enzymes. For instance, changes in liver blood flow, enzyme expression levels, or organ volumes can impact metabolic ratios.

4. **Explore Model Assumptions:**
   - Review the assumptions made in the model regarding the metabolic pathways and their kinetics. Adjusting these assumptions may lead to higher metabolic ratios.

5. **Parameter Sensitivity Analysis:**
   - Conduct a sensitivity analysis on the model parameters related to metabolism to identify which parameters have the most significant impact on the metabolic ratio. This can help in pinpointing the key determinants of metabolic exposure.

6. **Model Validation:**
   - Validate the model by comparing the simulated metabolic ratios with available in vivo or in vitro data. Adjust the model parameters to better match the observed metabolic ratios.
